projects/storefrontapp-e2e-cypress/cypress/helpers/cart.ts

```diff
@@ -69,51 +82,71 @@ function goToProductFromSearch(id: string, mobile: boolean) {
   }
 }
 
-export function addProductToCartViaAutoComplete(mobile: boolean) {
-  const product = products[0];
+function checkMiniCartCount(expectedCount) {
+  return cy.get('cx-mini-cart').within(() => {
+    cy.get('.count').should('contain', expectedCount);
+  });
+}
 
-  goToProductFromSearch(product.code, mobile);
+export function validateEmptyCart() {
+  cy.get('cx-breadcrumb h1').should('contain', 'Your Shopping Cart');
+  cy.get('.EmptyCartMiddleContent').should(
+    'contain',
+    'Your shopping cart is empty'
+  );
+}
 
+export function addToCart() {
   cy.get('cx-add-to-cart')
     .getAllByText(/Add To Cart/i)
     .first()
     .click({ force: true });
-  cy.get('cx-added-to-cart-dialog [aria-label="Close"]').click({ force: true });
+}
 
-  const miniCart = cy.get('cx-mini-cart');
-  miniCart.within(() => {
-    cy.get('.count').should('contain', 1);
-  });
-  miniCart.click({ force: true });
+export function closeAddedToCartDialog() {
```

Comment:
You don't understand some part?
Ask about it. Don't skip it.
It often means that code is not readable or business logic is only known by some people.